packages/calypso-e2e/src/lib/components/editor-settings-sidebar-component.ts

```diff
@@ -94,20 +124,16 @@ export class EditorSettingsSidebarComponent {
 	 *
 	 * If the section is already open, this method will pass.
 	 *
-	 * @param {ArticleSections} name Name of section to be expanded.
-	 * @returns {Promise<void>} No return value.
+	 * @param {string} name Name of section to be expanded.
 	 */
-	async expandSection( name: ArticleSections ): Promise< void > {
+	async expandSection( name: string ): Promise< void > {
 		if ( await this.targetIsOpen( selectors.section( name ) ) ) {
 			return;
 		}
 
-		// Avoid the wpcalypso/staging banner.
-		await this.scrollToBottomOfSidebar();
-
 		const editorParent = await this.editor.parent();
 		const sectionLocator = editorParent.locator( selectors.section( name ) );
-		await sectionLocator.click();
+		await sectionLocator.click( { position: { x: 5, y: 5 } } );
```

Comment:
This effectively removes the need to call `scrollToBottomOfSidebar`, which hard-coded the expectation that a section called `Discussion` will be the last item in the settings panel.

That would have been brittle, plus for Jetpack settings panel, this won't work because the `Discussion` section does not exist there. So instead of forcing a scroll to the bottom, click on the bounding box at x:5 and y:5 to avoid the staging/wpcalypso badge entirely.

Reply:
This is such a great idea!! 🥳